buildenv/jenkins/JenkinsfileBase

```diff
@@ -875,6 +875,8 @@ def post(output_name) {
 		//call the archive function for each file
 		archiveFile("aqa-tests/testenv/testenv.properties", true)
 		archiveFile("aqa-tests/TKG/**/*.tap", true)
+		archiveFile("jvmtest/perf/odm8100eGa/leftoverResults/*/*.txt", false)
+		archiveFile("jvmtest/perf/odm8100eGa/leftoverResults/*/*.log", false)
```

Comment:
- `leftoverResults` dir is a general dir for perf logs. We should not limit it to `odm8100eGa`. 
-  we should archive everything under `leftoverResults` dir, not just *.txt and *.log files. 
-  `leftoverResults` dir only gets generated when running perf. We should have a check here.

This solution is not ideal.

Reply:
Hi Lan, I do have the concern here too, while the current jenkins script (on rt-performance repo) has created this leftoverResults folder specifically into this path (under jvmtest/, which is another concern we might need to focus as well). 
- Do you have to move the leftoverResults into a more general level scope (not only with odm8100), make changes on script in rt-performance  first?
- yeah, I double checked the files inside this folder, only txt and log files moved into this folder, and we can use * to include them all
- Yeah, I do agree that we need a check, as you suggested on slack, and I tested that it doesn't generate extra warnings in the console logs, but I'll add it for sure.